src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -222,8 +223,8 @@ Please either install the missing font or choose another one.</value>
     <comment>{Locked="2023.5.31","MacType"}</comment>
   </data>
   <data name="RendererErrorOther" xml:space="preserve">
-    <value>Renderer encountered an unexpected error: {0:#010x} {1}</value>
-    <comment>{Locked="{0:#010x}","{1}"} {0:#010x} is a placeholder for a Windows error code (e.g. 0x88985002). {1} is the corresponding message.</comment>
+    <value>Renderer encountered an unexpected error: {0:#010x} {1} "{2}"</value>
+    <comment>{Locked="{0:#010x}","{1}"} {0:#010x} is a placeholder for a Windows error code (e.g. 0x88985002). {1} is the corresponding message. {2} is the filename.</comment>
   </data>

```

Comment:
Is there a chance that `parameter` will be empty for non-shader-related Renderer errors? If so, should we conditionally add the filename or parameter to the message, when we construct it?

Reply:
Sorry, I missed your edit! Don't worry too much about other languages - the automatic translation service should come through and automatically update the messages.

Your analysis makes sense of when and where the filename/parameters come from, as well. I appreciate it!

Reply:
Oh, I think I found what I was worried about!

There is one more call to `p.warningCallback` in `AtlasEngine.r.cpp`:

```
    if (_p.warningCallback)
    {
        try
        {
            _p.warningCallback(hr, {});
        }
        CATCH_LOG()
    }
```

If the render engine fails _for a non-shader-related_ reason, like DirectWrite ran out of memory or the graphics hardware did not respond in time, I think we will display an error message with empty quotes in it:

```
/!\ Error                                                                 [X]

The renderer encountered an unexpected error: 0x80070057 Invalid Parameter ""

                                                                       [ OK ]
```

Reply:
I see the issue now! I should've searched the codebase for all calls to `p.warningCallback`.
I made a commit to fix this, is the implementation fine?